simulator/docs/record-and-replay-cluster-events.md

```diff
@@ -0,0 +1,95 @@
+# [Beta] Record your real cluster's events and replay them in the simulator
+
+You can record events from your real cluster and replay them in the simulator. This feature is useful for reproducing issues that occur in your real cluster.
+
+## Record events
+
+To record events from your real cluster, you need to follow these steps:
+
+1. Set `true` to `recorderEnabled`.
+2. Set the path of the kubeconfig file for your cluster to `KubeConfig`.
+   - This feature only requires the read permission for events.
```

Comment:
Does it? Looks like recorder.go have an event handers though. I guess the read permissions for all resources to be recorded are required.

Reply:
> the read permissions for all resources to be recorded are required.

That's right.
"This feature only requires the read permission for resources." was correct.